docs/user-guide/cli-using-initializing-user-configuration.md

```diff
@@ -0,0 +1,68 @@
+# Initializing user configuration
+
+As an application developer or Zowe CLI user, you can manage your connection details efficiently and in one location.
+
+That could mean relying on a team configuration file, or creating your own *user* configuration file.
+
+To create your own user configuration, start with a global team configuration file that you have created or was provided to you. In this way, a single global configuration can become the basis for multiple user-specific configurations that are created with modifications tailored to particular requirements.
+
+To learn more about how profiles and configuration files work, see [How Zowe CLI uses configurations](../user-guide/cli-using-understand-profiles-configs.md).
+
+
+**[is there a use case for a team leader to create a user config file for *someone else*? We address app developers in the first graf. But I also want to make sure that we don't need to add instructions or best practices or context in this top section for team leaders.]**
+
+## Creating user profiles
+
+Generate a *user* configuration file that overrides the values defined in the global `zowe.config.json` file:
+
+1. Open a command line prompt and issue the following command to generate a global team configuration file:
+
+    ```
+    zowe config init --global-config
```

Comment:
for creating a user config in the Zowe home directory the command is:
`zowe config init --user-config --global-config`